src/main/kotlin/lotto/Lottos.kt

```diff
@@ -0,0 +1,7 @@
+package lotto
+
+class Lottos(private val _lottos: List<Lotto>) {
+
+    val get: List<Lotto>
+        get() = _lottos.toList()
+}
```

Comment:
```suggestion
class Lottos(val lottos: List<Lotto>)
```
내부에서 _lottos를 사용하지 않네요.
이렇게 사용해도 될 것 같아요.

Reply:
해당 lottos 를 방어적 복사를 하기 위해서 이렇게 사용해보았는데요~
제안 주신 것 처럼 사용하면 코틀린에서 별도의 .toList() 없이 방어적 복사를 만들어 get 프로퍼티를 제공해주나요?

Reply:
Lottos 클래스 내부에서 컬렉션을 변경하지 않고 그대로 캡슐화해서 사용하고 있기 때문에, 이 상황에서는 방어적 복사를 사용할 필요가 없습니다 😄